ssr-express/spec/lib/AmpSsrMiddlewareSpec.js

```diff
@@ -59,5 +60,30 @@ describe('Express Middleware', () => {
           expect(result).toEqual('original');
         });
     });
+
+    it('Skips Resource Requests', () => {
+      const staticResources = ['/image.jpg', '/image.svg', '/script.js', '/style.css'];
```

Comment:
tip: you can dynamically generate specs:

```
['/image.jpg', '/image.svg', '/script.js', '/style.css'].forEach(url => {
  it(url, () => {
      runMiddlewareForUrl(middleware, url)
           .then(result => {
             expect(result).toEqual('original');
           });
  });
});
```

Reply:
This is great. Rewrote.